acme/acme/client_test.py

```diff
@@ -121,6 +121,26 @@ def test_update_registration(self):
         self.assertRaises(
             errors.UnexpectedUpdate, self.client.update_registration, self.regr)
 
+    def test_deactivate_account(self):
+        self.response.headers['Location'] = self.regr.uri
+        self.response.json.return_value = self.regr.body.to_json()
+        self.assertEqual(self.regr, self.client.deactivate(self.regr))
+
+    def test_deactivate_account_bad_status(self):
+        self.response.headers['Location'] = self.regr.uri
+        self.response.json.return_value = self.regr.body.to_json()
+        self.response.status_code = 500  # something that isn't 200 or 202
+        self.assertRaises(
+            errors.DeactivationError,
+            lambda: self.client.deactivate(self.regr))
```

Comment:
I usually prefer lambda's since they seem more explicit. But your style is indeed more consistent with the codebase.